core/Pos/Core/Genesis/Types.hs

```diff
@@ -135,16 +142,58 @@ mkGenesisDelegation psks = do
         throwError "one of the delegates is also an issuer, don't do it"
     return $ UnsafeGenesisDelegation resMap
 
+data TestBalanceOptions = TestBalanceOptions
+    { tsoPoors        :: Word
+    , tsoRichmen      :: Word
+    , tsoRichmenShare :: Double
+    , tsoTotalBalance :: Word64
+    } deriving (Show)
+
+data FakeAvvmOptions = FakeAvvmOptions
```

Comment:
In what sense is this "fake"? Could we call it `TestnetAvvmOptions` instead?

Reply:
In a sense generated certificates aren't real

Reply:
Would it not be OK to use the same type for "real" and "fake"? They have the same form, no? Within the program the testnet and mainnet avvm certs are equally real.